src/main/kotlin/com/mineinabyss/launchy/data/Dirs.kt

```diff
@@ -27,9 +31,13 @@ object Dirs {
     val configFile = config / "mia-launcher.yml"
     val versionsFile = config / "mia-versions.yml"
 
+    @OptIn(DelicateCoroutinesApi::class)
     fun createDirs() {
         config.createDirectories()
-        mineinabyss.createDirectories()
+        if (mineinabyss.notExists()) {
+            mineinabyss.createDirectories()
+            GlobalScope.launch { FabricInstaller.installBaseConfigs() }
```

Comment:
I don't think installing configs should be part of creating directories, this should be done alongside all other mod downloads (that way you don't need to use GlobalScope.launch too). You will probably need to track what version of the zip has been installed in config, not sure what the best way of doing that would be though.

Reply:
could just make it download from a configlink on each mod?
so if its different itll know to download again, like mods themselves do?

Reply:
Thing is tho if we update something I didnt wanna overwrite peoples potential changes.
Hence it only does stuff rn if the directory is being created.
Means people would have to delete the entire directory ofc if we change stuff so

best way might be to download for each separate mod unless a config for it already exists?

Reply:
I think scyu had some ideas for how we should be installing things in an isolated way, maybe chat with him about that and how configs would tie into it.

Reply:
Ah I see you mean beyond that with like people's own modifications? I guess we just shouldnt expect any configs to be modifiable. I think having an option to toggle config per mod is better because of this, since people could disable the auto updates only for config and use whatever they want themselves

Reply:
Is that better? rn it downloads and unzips upon downloading the mod.